OpenTESArena/src/Entities/Camera3D.cpp

```diff
@@ -6,8 +6,8 @@
 #include "../Math/Constants.h"
 #include "../Math/Quaternion.h"
 
-Camera3D::Camera3D(const Double3 &position, const Double3 &direction)
-	: position(position), forward(direction)
+Camera3D::Camera3D(const Double3 &direction, const Double3 &position)
```

Comment:
I actually prefer having `position` first in the parameter list. The 3D frame (forward, right, up) is private because those values should only be changed through methods, but the position can be changed directly, so it is public.